### PR TITLE
Intergration test addition and migration

### PR DESCRIPTION
**Pull request by yasassri**
This PR contains the latest tests for HTTP and SQL connector.

### CODE REVIEW

File: src/test/java/org/ballerinalang/client/HttpTestClient.java [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InterruptedIOException;
+import java.net.UnknownHostException;
+import java.util.HashMap;
+import javax.net.ssl.SSLException;
+
+/**
+ * HttpRequest sending client
```
Comment (lafernando) [inline src/test/java/org/ballerinalang/client/HttpTestClient.java]: Main Javadoc comment must end with a full stop.

File: src/test/java/org/ballerinalang/client/HttpTestClient.java [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+    public void setContentType(String contentType) {
+        this.contentType = contentType;
+    }
+
+    public void setHeader(String header, String value) {
+        this.requestHeaders.put(header, value);
+    }
+
+    /**
+     * @param resourcePath
```
Comment (lafernando) [inline src/test/java/org/ballerinalang/client/HttpTestClient.java]: If putting the Javadoc comment, it must be defined properly. The main comment must be here, and parameter definitions and the return value. Fix in all the places.

File: src/test/java/org/ballerinalang/client/ResponseHolder.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
+package org.ballerinalang.client;
```
Comment (lafernando) [inline src/test/java/org/ballerinalang/client/ResponseHolder.java]: Add license header, fix in all the places.

File: src/test/java/org/ballerinalang/tests/connectors/http/ContentEncodingTests.java [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotNull;
+
+/**
+ * Test class for testing transfer and content encoding
+ * Uses EncodingService.bal
+ */
+public class ContentEncodingTests extends BallerinaBaseTest {
+
```
Comment (lafernando) [inline src/test/java/org/ballerinalang/tests/connectors/http/ContentEncodingTests.java]: Remove random whitespace, fix in all the places.

File: src/test/java/org/ballerinalang/tests/connectors/http/ContentEncodingTests.java [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+        ResponseHolder response = request.postRequest("/encoded/spayload", payload);
+
+        GzipCompressor compressor = new GzipCompressor();
+        String decompressedMessage = compressor.decompress(new ByteArrayInputStream(response.getResponseAsByteArray()));
+        assertEquals(decompressedMessage, payload);
+        assertEquals(response.getResponseCode(), 200);
+
+    }
+
+    //    @Test public void chunkedMessageForPut() throws Exception {
```
Comment (lafernando) [inline src/test/java/org/ballerinalang/tests/connectors/http/ContentEncodingTests.java]: Remove commented code. If a test is disabled temporarily, use the annotation to set it to disabled.

File: src/test/java/org/ballerinalang/tests/connectors/http/HTTPMethodsTests.java [hunk 6]
```diff
@@ -31,6 +35,7 @@
     @Test(description = "Test all methods in one server resource") public void testAllVerbsInOneResource()
             throws IOException {
```
Comment (lafernando) [inline src/test/java/org/ballerinalang/tests/connectors/http/HTTPMethodsTests.java]: Don't put a blank line before starting a method definition, fix in all the places.

File: src/test/java/org/ballerinalang/tests/connectors/rdbms/operations/delete/DBDeleteTest.java [hunk 7]
```diff
@@ -0,0 +1,252 @@
...
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import static org.testng.Assert.assertEquals;
+
+/**
+ * Tests database delete operations and its variations
+ * Uses DeleteTestService.bal service and DBDeleteTest.bal
+ */
+
```
Comment (lafernando) [inline src/test/java/org/ballerinalang/tests/connectors/rdbms/operations/delete/DBDeleteTest.java]: Remove blank line.

File: src/test/java/org/ballerinalang/client/HttpTestClient.java (hunk 1 above)
Comment (yasassri) [reply src/test/java/org/ballerinalang/client/HttpTestClient.java]: This is fixed with https://github.com/ballerinalang/integration-tests/pull/10

File: src/test/java/org/ballerinalang/client/HttpTestClient.java (hunk 2 above)
Comment (yasassri) [reply src/test/java/org/ballerinalang/client/HttpTestClient.java]: This is fixed with #10

File: src/test/java/org/ballerinalang/client/ResponseHolder.java (hunk 3 above)
Comment (yasassri) [reply src/test/java/org/ballerinalang/client/ResponseHolder.java]: This is fixed with #10

File: src/test/java/org/ballerinalang/tests/connectors/http/ContentEncodingTests.java (hunk 4 above)
Comment (yasassri) [reply src/test/java/org/ballerinalang/tests/connectors/http/ContentEncodingTests.java]: This is fixed with #10

File: src/test/java/org/ballerinalang/tests/connectors/http/ContentEncodingTests.java (hunk 5 above)
Comment (yasassri) [reply src/test/java/org/ballerinalang/tests/connectors/http/ContentEncodingTests.java]: This is fixed with #10

File: src/test/java/org/ballerinalang/tests/connectors/http/HTTPMethodsTests.java (hunk 6 above)
Comment (yasassri) [reply src/test/java/org/ballerinalang/tests/connectors/http/HTTPMethodsTests.java]: This is fixed with #10

File: src/test/java/org/ballerinalang/tests/connectors/rdbms/operations/delete/DBDeleteTest.java (hunk 7 above)
Comment (yasassri) [reply src/test/java/org/ballerinalang/tests/connectors/rdbms/operations/delete/DBDeleteTest.java]: This is fixed with #10

**Pull request merged**